### DISCUSSION TITLE
Verification link problems - Unable to verify my Owncast site

### DISCUSSION THREAD

**Discussion by deadtom76** [mastodon/mastodon#23040] [Q&A]
So I'm trying to add my Owncast site to the list of verified sites on my profile on Fosstodon.

- The Owncast site is https
- rel="me" is in the link
- The link is not inside an iframe
- I am removing the link from my profile, saving, adding the link, and saving again each time I try

Doing a curl -4 https://owncast.deadtom.me | grep rel=\"me\" returns the following:

```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
10    <link rel="me" href="https://fosstodon.org/@deadtom">0:02 --:--:--     0
0  7751  100  7751    0     0   2722      0  0:00:02  0:00:02 --:--:--  2721
```

I would assume that this means Mastodon should be able to pick it up, but it's not happening.

The one caveat, is that my Owncast site is behind a reverse proxy.

Could that potentially be the problem?

Thanks.

**Comment by rbairwell** [comment]
I've just tried looking into this for you, and I can't see a "rel="me" link on that site and `curl -4 https://owncast.deadtom.me/ | grep rel="me"` (curl 7.68.0 on Linux) gives no matching text (searching that page's raw source for fosstodon manually provides no results: but I think this might be because the CMS that OwnCast is providing only populates that part of the page via Javascript).

Can you try putting the link back in the headers of that page, removing it from your Mastodon profile page, waiting a few minutes, readd it and then wait around ten minutes to see if it makes a difference (and if https://mastodon-link-debugger.vercel.app/profile/@deadtom@fosstodon.org gives any clue as to why).

**Comment by deadtom76** [comment]
Oh, right. I upgraded Owncast last night, and it probably removed the link. I just put it back in, restarted owncast, and removed and re-added the link to Mastodon.

curl -4 https://owncast.deadtom.me | grep 'rel="me"' shows it there now, and it's in the source code in Firefox.

The link debugger shows that rel="me" is not there, but it is. I wonder if there is something in the Owncast CSS that is overriding that.

**Comment by deadtom76** [comment]
More information:

The link that was getting picked up by the link debugger was the one in the left hand column, generated by a script. 

Edit: The rel=me is in the link generated by the script, but it's not in `<a>` or `<link>` form, so Mastodon doesn't pick it up. Mastodon doesn't see the link under `<head>` at all.
